src/app/settings/views/Configuration/Security/Security.test.tsx

```diff
@@ -7,16 +7,23 @@ import {
   rootState as rootStateFactory,
   tlsCertificate as tlsCertificateFactory,
   tlsCertificateState as tlsCertificateStateFactory,
+  controller as controllerFactory,
+  controllerState as controllerStateFactory,
+  vaultEnabledState as vaultEnabledStateFactory,
 } from "testing/factories";
 import { renderWithBrowserRouter } from "testing/utils";
 
-it("displays loading text if TLS certificate has not loaded", () => {
+it("displays loading text if TLS certificate or Vault Status has not loaded", () => {
```

Comment:
You're checking whether the loading spinner is displayed when _both_ TLS certificate and Vault Status haven't loaded. Might be worth breaking down to separate tests instead of modifying the existing one to make sure that loading either of them adds the loading indicator.